components/more-less/more-less.js

```diff
@@ -198,18 +198,19 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 
 	__adjustToContent_makeActive() {
 		this.inactive = false;
-		this.__maxHeight = this.height;
+		// Include 1px of given room to account for issues with rounding the content's scroll height
+		this.__maxHeight = `${this.height + 1}px`;
```

Comment:
I think this is actually "fixing" it by somewhat "breaking" `__maxHeight` (and therefore breaking the cycle). Unlike the two cases below, `height` is a string (`4em`) so adding `1` to it creates `4em1px`, which the browser just kinda throws away. I do think I've found a way to reproduce in `core` demos though, so I can put that up tomorrow.